src/components/MoneyReportHeaderStatusBar.tsx

```diff
@@ -25,9 +36,11 @@ function MoneyReportHeaderStatusBar({nextStep}: MoneyReportHeaderStatusBarProps)
     return (
         <View style={[styles.dFlex, styles.flexRow, styles.alignItemsCenter, styles.overflowHidden, styles.w100, styles.headerStatusBarContainer]}>
             <View style={[styles.mr3]}>
-                <Badge
-                    text={translate(nextStep.title === CONST.NEXT_STEP.FINISHED ? 'iou.finished' : 'iou.nextStep')}
-                    badgeStyles={styles.ml0}
+                <Icon
+                    src={Object.values(CONST.NEXT_STEP.ICONS).includes(nextStep?.icon) ? iconMap[nextStep.icon] : iconMap[CONST.NEXT_STEP.ICONS.HOURGLASS]}
```

Comment:
Let's roll this back to  `src={iconMap[nextStep.icon] || Expensicons.Hourglass}`, it's much more readable, apologies for the hassle